src/snowflake/snowpark/modin/plugin/PANDAS_CHANGELOG.md

```diff
@@ -46,6 +46,7 @@
 
 ### New Features
 - Added partial support for `SeriesGroupBy.apply` (where the `SeriesGrouBy` is obtained through `DataFrameGroupBy.__getitem__`).
```

Comment:
Should we use `cache_result` similar to Snowpark? I'd be even fine to define an alias `cache` and `cache_result` and add both to API docs.